Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -120,6 +120,11 @@ class BrowserViewController: UIViewController {
     var pendingDownloadURL: URL?
 
     let downloadQueue = DownloadQueue()
+    
+    fileprivate let shieldBlockStats = ShieldBlockedStats().then { _ in
+        Domain.loadShieldsIntoMemory {}
```

Comment:
The main thing here is that this information is so universal that it needs to supersede tabs. Most adblocking is tab specific, as rules need to apply to each `WKWebView` instance.

This may move when I get deeper into the adblock implementations and stats collections, but not really sure where else to put it right now, since so many pieces are missing currently.